src/components/play/PlayUI.js

```diff
@@ -99,12 +142,32 @@ class PlayUI extends Component {
     }
 
     async componentDidUpdate(prevProps) {
+        const { round, user, display, setIsRecordingSequence } = this.props
+        const oldRound = prevProps.round
+        let redraw = false
+        let shouldRecalculateParts = false
+        const _this = this
```

Comment:
why do we need this?

Reply:
Looks like something the previous developer was fond of doing, I'm not sure why.